packages/cli/src/commands/generate/cell/cell.js

```diff
@@ -124,6 +126,9 @@ export const files = async ({
     webPathSection: REDWOOD_WEB_PATH_NAME,
     generator: 'cell',
     templatePath: `mock${templateNameSuffix}.js.template`,
+    templateVars: {
+      mockIdValue,
```

Comment:
```suggestion
      mockIdValue1: idType === 'String' ? "'42'" : 42,
      mockIdValue2: idType === 'String' ? "'43'" : 43,
      mockIdValue3: idType === 'String' ? "'44'" : 44,
```

Reply:
Oh, I was thinking of passing an array of values and updating the list template, but if you are ok with three variables, then works for me.

Hm. Yeah I think I will change to an array.